canonical-web-and-design/ubuntu.com
# Copy update -  security/certifications

## Done

- Update text in FIPS 140-2 section
- Add 'FIPS certification and CIS compliance with Ubuntu' webinar section

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/certifications
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page matches [copy doc](https://docs.google.com/document/d/1CiF-dPqG1Y--h4FWfgKB1ft4G3nC2Boa8NT_JG7D6dQ/edit#heading=h.kp4ajk60s1gx)


## Issue / Card

Fixes #8351 

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/94447781-17c44300-01a2-11eb-882f-f1a4c2441f97.png)
